### ISSUE TITLE
How to change color scheme?

### ISSUE THREAD

**Issue by amarakon** [dabisu/sakura#7]
I have these lines in my sakura.conf:
colorset1_fore=rgb(216,222,233)
colorset1_back=rgb(46,52,64)
colorset1_curs=rgb(216,222,233)
colorset1_scheme=1
colorset1_key=F1
last_colorset=1

But these only change the foreground background and cursor color. I want to change all the colors. Can I change the entire color scheme?

**Comment by dabisu** [comment]
No, you can't. Currently in sakura only fore/back and cursor color are configurable. The rest of colors are fixed in the palettes included in sakura.

**Comment by rmb122** [comment]
In actually, you can modify color palette by [OSC sequences](https://en.wikipedia.org/wiki/ANSI_escape_code#OSC_(Operating_System_Command)_sequences). You can get more information in https://github.com/lemnos/theme.sh/blob/master/src/theme.sh#L248
